From c8c16c02346f5cdd1035454b575af82648124cc4 Mon Sep 17 00:00:00 2001
From: Ziad Elhanafy <ziad.elhanafy@arm.com>
Date: Wed, 8 Nov 2023 14:54:08 +0000
Subject: [PATCH] rss/kronos: Map SI CL1 <-> RSS MHU Interrupts in GIC multi
 view

Map the transmit and receive MHU interrupts between Safety
Island Cluster 1 and RSS in the GIC multi view.

Upstream-Status: Pending
Signed-off-by: Ziad Elhanafy <ziad.elhanafy@arm.com>
---
 platform/ext/target/arm/rss/kronos/fainlight_gic_lib.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/platform/ext/target/arm/rss/kronos/fainlight_gic_lib.c b/platform/ext/target/arm/rss/kronos/fainlight_gic_lib.c
index 77e84150d..837717915 100644
--- a/platform/ext/target/arm/rss/kronos/fainlight_gic_lib.c
+++ b/platform/ext/target/arm/rss/kronos/fainlight_gic_lib.c
@@ -110,6 +110,8 @@ static struct gic_mv_i2v_map view_spi_map[] = {
     { 116, 2 }, /* MHU CL1 To CL0 Sender */
     { 118, 2 }, /* MHU CL1 To CL2 Sender */
     { 123, 2 }, /* MHU CL2 To CL1 Receiver */
+    { 124, 2 }, /* MHU to RSS Cluster 1 S */
+    { 125, 2 }, /* MHU from RSS Cluster 1 S */
     { 128, 2 }, /* FMU */
     { 129, 2 }, /* SSU */
     { 134, 2 }, /* FMU Non-Critical */
-- 
2.25.1

